src/isort/format.rs

```diff
@@ -72,7 +76,17 @@ pub fn format_import_from(
         }
     }
 
-    format_multi_line(import_from, comments, aliases, is_first)
+    if force_single_line {
+        aliases
+            .iter()
+            .enumerate()
+            .map(|(i, _)| {
+                format_single_line(import_from, comments, &aliases[i..=i], is_first && i == 0).0
+            })
+            .join("")
```

Comment:
I wonder if we could instead model this as an additional transform from `OrderedImportBlock` to `OrderedImportBlock`, and not change these format functions. Something like `split_imports`?

Reply:
Interesting, let me look into that.